Fix #176 -- Allow overriding the img-tag dimensions attributes

atnartur opened the pull request:
Closes #176

codingjoe commented:
Please let me know if you need help with the tests. I believe it's only the attribute order that changed, should be pretty straightforward. However, I would appreciate it if you could add a new test to prevent regression.

atnartur commented:
Thank you very much for your support and advice! I added a test to check new attributes and also fixed `alt="None"` value supported with the corresponding test.

I hope, everything will be okay in CI this time.

codingjoe reviewed (approved):
Thanks, @atnartur, I took the liberty to amend some changes, like dropping a leftover print statement.
However, I want to not, that I also reverted your changes to the `alt`-attribute. Please beware that setting an `alt` attribute is required. There are cases, where an empty string is the best way to go, but that would be a matter for a separate patch.
